HousingSearchApi/V1/Domain/Identification.cs

```diff
@@ -0,0 +1,19 @@
+using Newtonsoft.Json;
+
+namespace HousingSearchApi.V1.Domain
+{
+    public class Identification
+    {
+        [JsonProperty("identificationType")]
```

Comment:
Do we need all of these JsonProperty attributes (here and on the other models) or can we just setup the JsonSerialiser to use camel case by default?

Reply:
Thought about doing so - I'll fix it in my next ticket.